addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -735,19 +742,41 @@ blockquote {
             $-btn-primary-border-color: if($-btn-primary-border, o-color($-btn-primary-border), $-btn-primary-color);
             .btn-fill-primary {
                 @include button-variant($-btn-primary-color, $-btn-primary-border-color);
+                @if $-btn-primary-gradient {
+                    background-image: $-btn-primary-gradient;
+                    &:hover {
+                        @extend .btn-custom:hover;
+                    }
+                } @else {
+                    // Cancel parent o_cc's button gradient to make sure this
+                    // o_cc's button defined solid color is displayed.
+                    background-image: none;
+                }
```

Comment:
Mmh, maybe this should be inside the `.o_cc {` rule above? That would make it a unique rule instead of one per preset if I am not mistaken?

Reply:
Indeed - actually my old approach was incomplete because it did not cover the same situation for outline buttons. The global solution at `.o_cc` level does cover all types of buttons.